org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/device/MonitorView.java

```diff
@@ -116,6 +124,24 @@ public void run() {
 		
 		ViewUtil.addGroups("refresh", mans, refresh);
 	}
+	
+	
+	private IAction createPreferenceAction(String label, String preference, String icon) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createPreferenceAction" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)